siliconcompiler/core.py

```diff
@@ -848,7 +848,7 @@ def set(self, *args, field='value', clobber=True, step=None, index=None):
             self.error(e)
 
     ###########################################################################
-    def clear(self, *keypath):
+    def clear(self, *keypath, step=None, index=None):
```

Comment:
I missed api name change earlier. At some point we had discussed 'unset'. With the parmeter field/state being 'set' and the value of 'set' being True/False. Not sure clear is the right term here...very subjective. Both clear/unset are a bit since we are reverting to the default value. 

Bash : unset delets a variable during program execution
Hardware: set ==set/preset (1), clear/reset(0)
css=unset resets property to defaut value


If I had to choose, I would say that set()/unset() is a more natural coupling thatn set()/clear(), but perhaps others can weigh in.

@gadfort

Reply:
I agree with @aolofsson `unset` is a more natural pairing with `set`